.github/workflows/playwright.yml

```
@@ -144,9 +147,64 @@
       #   if: always()
       #   run: cat o2.log
 
+      - name: Sanitize Test Filename
+        shell: bash 
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:50: Double quote to prevent globbing and word splitting

(shellcheck)